### DISCUSSION TITLE
how to access the intern database from outside?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2009] [Help]
Hello,

I would need to allow an external service to reach the couchdb databse? Any idea how I can reach that?

Thank you

**Comment by shogunpurple** [comment]
Hi @oswaldB - how/where is your budibase instance hosted?

**Reply by oswaldB** [reply]
My budibase is hosted in a dcoker on gcp on its own serveur.

I followed your install script.

**Reply by shogunpurple** [reply]
You should be able to access couchDB at:

`<VM-IP>:10000/db/_utils`

**Reply by mjashanks** [reply]
This is now documented at https://docs.budibase.com/self-hosting/accessing-couchdb

**Comment by KernOil** [comment]
its here #1657

**Comment by oswaldB** [comment]
Hello,

I have an access denied.


But I am mostly interesting in having the connexion credentials.

So you know where I can find them in my server?


Thanks

**Reply by mike12345567** [reply]
Hi @oswaldB the password and username to access the database will be set in your docker environment variable file, so where you have your `docker-compose.yaml` you'll either have a `.env` or `hosting.properties` file, look in these for your couch DB username and password.

**Comment by oswaldB** [comment]
Thank you.
